pkg/bindinfo/global_handle.go

```diff
@@ -682,3 +685,47 @@ func (h *globalBindingHandle) Stats(_ *variable.SessionVars) (map[string]any, er
 	m[lastPlanBindingUpdateTime] = h.getLastUpdateTime().String()
 	return m, nil
 }
+
+// LoadBindingsFromStorageToCache loads global bindings from storage to the memory cache.
+func (h *globalBindingHandle) LoadBindingsFromStorage(sqlDigest string) (Bindings, error) {
+	if sqlDigest == "" {
+		return nil, nil
+	}
+	bindings, err, _ := h.syncBindingSingleflight.Do(sqlDigest, func() (any, error) {
+		return h.loadBindingsFromStorageInternal(sqlDigest)
+	})
+	if err != nil {
+		logutil.BgLogger().Warn("fail to LoadBindingsFromStorageToCache", zap.Error(err))
+		return nil, err
+	}
+	if bindings == nil {
+		return nil, nil
+	}
+	return bindings.(Bindings), err
+}
+
+func (h *globalBindingHandle) loadBindingsFromStorageInternal(sqlDigest string) (any, error) {
+	var bindings Bindings
+	selectStmt := fmt.Sprintf("SELECT original_sql, bind_sql, default_db, status, create_time, update_time, charset, collation, source, sql_digest, plan_digest FROM mysql.bind_info where sql_digest = '%s'", sqlDigest)
+	err := h.callWithSCtx(false, func(sctx sessionctx.Context) error {
+		rows, _, err := execRows(sctx, selectStmt)
```

Comment:
you can directly pass the parameters as args inside here, rather than do the escape yourself.

Reply:
Done